src/main/java/oshi/hardware/platform/linux/LinuxGlobalMemory.java

```diff
@@ -124,7 +124,7 @@ public long getTotal() {
     }
     
     @Override
-    public long getSwapAvailable() {
+    public long getSwapUsed() {
         long swapAvailable = 0;

```

Comment:
You changed the method name but not the internals.  Also, need to change (or commit if you changed already) the updated SystemInfoTest.